app/utils/wiki-handlers/article.js

```diff
@@ -1,4 +1,4 @@
-import fetch from '../mediawiki-fetch';
+import fetch from 'fetch';
```

Comment:
Yeah, it already was going through Fastly so nothing changes on production where this call matters, other than it might fail on sandboxes, but that's OK from our perspective for this one case. `mediawiki-fetch` was only going through the proxy if the `internalCache` option was set, which only the user model was doing, so everything else was going through Fastly. We also do this tracking call client side on Desktop, so going through Fastly for the lyrics tracking call should be OK, and it seemed like a bit more refactoring was required to use the service here, so I thought it was best to leave it as is for now.